rust/main/agents/scraper/src/db/generated/gas_payment.rs

```diff
@@ -90,10 +87,6 @@ impl ColumnTrait for Column {
 impl RelationTrait for Relation {
     fn def(&self) -> RelationDef {
         match self {
-            Self::Destination => Entity::belongs_to(super::domain::Entity)
-                .from(Column::Destination)
-                .to(super::domain::Column::Id)
-                .into(),
             Self::Domain => Entity::belongs_to(super::domain::Entity)
                 .from(Column::Domain)
                 .to(super::domain::Column::Id)
```

Comment:
I know we were planning on deprecating `Domain` but I'm not sure why `Destination` is being removed from here

Reply:
It looks like we are getting gas payment which pay for messages in chain we don't support. Such chains are not mentioned in `domain` table. So, we cannot apply a foreign key constrained to `destination` field.